kroxylicious-additional-filters/kroxylicious-multitenant/src/test/resources/io/kroxylicious/proxy/filter/multitenant/OffsetCommit.test.yaml

```diff
@@ -9,7 +9,7 @@
   request:
     payload:
       groupId: mygroup
-      generationId: 0
+      generationIdOrMemberEpoch: 0
```

Comment:
Puzzled me for a while - doesn't this break wire compatibility?   No - this is just a field rename and field names play no part in the bytes on the wire.   Field order/type etc are unchanged.   So the only reason it affects Kroxylicious MT is because the test data knows the field names.